modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1722,31 +1837,37 @@ Besides the default `--help`, `-f`/`--fields` and `-o`/`--output` options, this
 > runtime-mgr standalone-application restart [options] <identifier>
 ----
 
-This command restarts the standalone (on-premises) application passed in `identifier`. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command restarts the standalone (on-premises) application passed in `identifier`. 
+
+// CLI DEFAULTS
+include::partial$cli-default-options.adoc[tag=CLIdefaultOptions]
 
 === runtime-mgr standalone-application stop
 
 ----
 > runtime-mgr standalone-application stop [options] <identifier>
 ----
 
-This command stops the standalone (on-premises) application passed in `identifier`. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command stops the standalone (on-premises) application passed in `identifier`.
+
+// CLI DEFAULTS
+include::partial$cli-default-options.adoc[tag=CLIdefaultOptions]
 
 === runtime-mgr standalone-application copy
 
 ----
 > runtime-mgr standalone-application copy [options] <source> <target> <targetIdentifier>
 ----
 
-This command copies the standalone (on-premises) application passed in `source` to the target passed in `target` and the server, server group or cluster ID or Name passed in `targetIdentifier`. +
+This command copies the standalone (on-premises) application passed in `source` to the target passed in `target` and the server, server group or cluster ID or Name passed in `targetIdentifier`. 
```

Comment:
Should it be "... server group or cluster ID, or Name passed ..."?

Reply:
Nope - it should be "name"